easybuild/easyblocks/w/wps.py

```diff
@@ -100,6 +100,10 @@ def configure_step(self):
         # netCDF dependency check + setting env vars (NETCDF, NETCDFF)
         set_netcdf_env_vars(self.log)
 
+        netcdff = get_software_root('netCDF-Fortran')
+        if netcdff:
+            env.setvar('NETCDFF', netcdff)
```

Comment:
Is this needed at all? It is supposed to be set in https://github.com/easybuilders/easybuild-easyblocks/blob/master/easybuild/easyblocks/n/netcdf.py#L132 already. I don't know why you need that in your case, the code looks the same to me

Reply:
you're right, just tested this and updated the PR without it.